--- a/switch/Library/PlatformBootManagerLib/PlatformBootManagerLib.inf
+++ b/switch/Library/PlatformBootManagerLib/PlatformBootManagerLib.inf
@@ -33,14 +33,21 @@
 
 [Sources]
   PlatformBm.c
+  SwitchBsdAutoBoot.c
+  SwitchBsdUsbHost.h
+  SwitchBsdUsbHost.c
+  SwitchBsdI2c.c
 
 [Packages]
   MdeModulePkg/MdeModulePkg.dec
   MdePkg/MdePkg.dec
   ShellPkg/ShellPkg.dec
+  NintendoSwitchPkg/NintendoSwitch.dec
 
 [LibraryClasses]
   BaseLib
+  IoLib
+  TimerLib
   BaseMemoryLib
   BootLogoLib
   CapsuleLib
@@ -78,6 +85,11 @@
   gUefiShellFileGuid
 
 [Protocols]
+  gEfiBlockIoProtocolGuid
+  gTegraPinMuxProtocolGuid
+  gTegra210ClockManagementProtocolGuid
+  gTegraUBootClockManagementProtocolGuid
+  gPmicProtocolGuid
   gEfiDevicePathProtocolGuid
   gEfiGraphicsOutputProtocolGuid
   gEfiLoadedImageProtocolGuid
